### ISSUE TITLE
Option to open wallet URL from command line

### ISSUE THREAD

**Issue by mflaxman** [blockcypher/bcwallet#31]
This might work:
https://pythonconquerstheuniverse.wordpress.com/2010/10/16/how-to-open-a-web-browser-from-python/

**Comment by mflaxman** [comment]
I'm having second thoughts on this feature.

The idea is that copy-pasting the block explorer URL for your xpub is annoying (mouse!) so I could add a an option to the menu that opens your default web browser with that URL. My concern is that doing that puts your xpub in your browser (and browser history), which some people have privacy concerns about. I think 90% of users won't care about this, but I could see for the 10% who do that they weren't clear what that button did and then they get angry that it does this and it's irreversible.

Perhaps better to stick with the current implementation and just force people to use their mouse to copy-paste if that's what they want to do... What do you think @matthieu or @acityinohio ?

**Comment by acityinohio** [comment]
Yeah, I see where you're coming from @mflaxman...and normally I hate to design for a vocal minority, but I imagine there would be all manner of folks that would cry wolf about this.

As a compromise, how about "copy link to clipboard"? Would make things more convenient, give folks an opportunity to inspect what's copied, but without directly exposing browser history. Obviously you have to hope that the clipboard is less leaky than a browser...which may or may not be true...

Maybe using pbcopy or xsel or something? https://stackoverflow.com/questions/7606062/is-there-a-way-to-directly-send-a-python-output-to-clipboard

**Comment by mflaxman** [comment]
Good idea @acityinohio, thanks for the feedback.

**Comment by jshoyes** [comment]
I made some edits to the code in regards to this issue, and wanted to push it to the repository. But I get the following message:
`fatal: Authentication failed for 'https://github.com/blockcypher/bcwallet.git/'`

Is there something else I should be doing?

**Comment by mflaxman** [comment]
Glad you're contributing to bcwallet @ShaunHoyes!

You'll need to fork the project and submit a push your changes to your own branch, and then submit the PR. Details here:
https://help.github.com/articles/using-pull-requests/
